REPOSITORIES.md

```diff
@@ -0,0 +1,35 @@
+## Repositories
+
+This document describes the lifecycle of repositories under the [Notary Project](https://github.com/notaryproject/) organization.
+
+Anyone can submit proposals regarding repositories' lifecycle by opening a GitHub issue in the
+[notaryproject/.github](https://github.com/notaryproject/.github) repository. The [Notary Project governance maintainers](MAINTAINERS) will take into account the community feedback and decide on the proposal.
+
+### Create a new sub-project
+
+A new sub-project can be created as a repository and contributed to the Notary Project organization by opening a GitHub issue to request [Notary Project governance maintainers](MAINTAINERS) to vote in the
+[notaryproject/.github](https://github.com/notaryproject/.github) repository. A new sub-project can be created after having a supermajority of approval from Notary Project governance maintainers.
+
+If the decision is to add the proposed project, then one of the Notary Project GitHub organization admins will assist the issue opener in creating a repository or transferring an existing repository to the Notary Project organization. Upon creation, the repository must be reviewed to make sure it follows [Notary Governance](GOVERNANCE.md). 
+
+### Lifecycle change 
+
+A repository can be archived if the sub-project maintainers no longer maintain it or no activity for a long time. The sub-project maintainers of a given repository can propose changing its status by opening a GitHub issue in the [notaryproject/.github](https://github.com/notaryproject/.github) repository. The [Notary Project governance maintainers](MAINTAINERS) will take into account the community feedback and decide on the proposal.
```

Comment:
* Please elaborate on what does ` sub-project maintainers no longer maintain it` means?
* ` or no activity for a long time.` Can we quantify how long?